pycord/ui/house.py

```diff
@@ -25,9 +25,8 @@
 from typing import Literal
 from uuid import uuid4
 
-from ..errors import ComponentException
-
 from ..enums import ButtonStyle, SelectMenuType
+from ..errors import ComponentException
 from ..media import Emoji
 from ..types import AsyncFunc
 from ..undefined import UNDEFINED, UndefinedType
```

Comment:
Reverts out of scope change.
```suggestion
from ..errors import ComponentException

from ..enums import ButtonStyle, SelectMenuType
from ..media import Emoji
from ..types import AsyncFunc
from ..undefined import UNDEFINED, UndefinedType
```